irondb/irondb.py

```diff
@@ -144,73 +218,29 @@ def get_intervals(self):
         return IntervalSet([Interval(0, int(time.time()))])
 
 
-class IronDBFinder(BaseFinder):
+class IRONdbFinder(BaseFinder):
```

Comment:
Changing this name is going to break existing installs.  Can we add an alias class to cover both names for backwards compatibility?